addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -129,6 +129,16 @@ export class DiscussCoreCommon {
                     }
                 }
             });
+            this.busService.subscribe("mail.shortcode/insert", (datas) => {
+                for (const data of datas) {
+                    this.store.CannedResponse.insert(data);
+                }
+            });
```

Comment:
You can pass array of data to `insert()`, so can just do `insert(datas)` without the `for-of` loop